parsedatetime/pdt_locales/en_A.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
```

Comment:
ah - that's a mistake then - the localeID should have been set to en_AU 

we will have to fix that and then bump the major version number IMO